### PR TITLE
Fixing redirects to work cross platform

### PR DESCRIPTION
**Pull request by robinbernon**
Fixing redirects to work cross platform

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -11,8 +11,8 @@ keywords = ["stdout", "stderr", "stdio", "redirect"]
 license = "MIT"
 
 [dependencies]
-libc = "0.2"
 tempfile = "3.0"
+filedescriptor = { version = "0.7.3", git = "https://github.com/robinbernon/wezterm", branch = "AddingStdRedirects" }
```
Comment (robinbernon) [inline Cargo.toml:15]: I'll change this to a fixed version number once my PR for filedescriptor gets through.

**Review by Stebalien — COMMENTED** [review]
Code generally looks good, I'll try to take a look at the underlying library this weekend.

File: Cargo.toml (hunk 1 above)
Comment (robinbernon) [reply Cargo.toml]: My PR went through. Filedescriptor is now using a public version number.

**Comment by robinbernon** [comment]
@Stebalien might be ready to merge now?

**Comment by robinbernon** [comment]
@Stebalien ?

**Comment by Stebalien** [comment]
Sorry, life is kind of crazy right now. I'll try to make some time this coming weekend.

But thanks for the reminder.

**Comment by robinbernon** [comment]
No worries fully understand, that's life! Take your time with this, no rush :)

**Review by Stebalien — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**